Neuron/Models/Realm/WalletModel.swift

```diff
@@ -24,4 +43,38 @@ class WalletModel: Object {
     override static func primaryKey() -> String? {
         return "address"
     }
+
+    var icon: WalleIconType {
+        get {
+            if let iconName = iconName {
+                return WalleIconType(rawValue: iconName)!
+            }
+            let iconList = WalleIconType.allType
+            var useCount = [Int].init(repeating: 0, count: WalleIconType.allType.count)
+            let realm = try! Realm()
+            realm.objects(WalletModel.self).forEach { (wallet) in
+                guard let idx = iconList.firstIndex(where: { $0.rawValue == wallet.iconName }) else { return }
+                useCount[idx] += 1
+            }
+            var minimumUsedCount = Int.max
+            var minimumUsedList = [WalleIconType]()
+            for (idx, count) in useCount.enumerated() {
+                if count < minimumUsedCount {
+                    minimumUsedCount = count
+                    minimumUsedList.removeAll()
+                }
+                if count == minimumUsedCount {
+                    minimumUsedList.append(iconList[idx])
+                }
+            }
+            let randomIdx = arc4random_uniform(UInt32(minimumUsedList.count))
```

Comment:
Prefer `UInt32.random(in:)`.

I also suggest we replace all `arc4random_uniform` with this.